# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/glib-networking/hotfix-path.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- glib-networking-2.80.1/proxy/libproxy/meson.build.vanilla	2026-09-09 16:27:40.746815794 +0300
+++ glib-networking-2.80.1/proxy/libproxy/meson.build	2026-09-09 16:27:50.348034601 +0300
@@ -7,7 +7,7 @@
   input: service + '.in',
   output: service,
   install: true,
-  install_dir: join_paths(datadir, 'dbus-1', 'services'),
+  install_dir: join_paths('share', 'dbus-1', 'services'),
   configuration: service_conf
 )
 
--- glib-networking-2.80.1/meson.build.vanilla	2026-09-09 16:26:15.913752511 +0300
+++ glib-networking-2.80.1/meson.build	2026-09-09 16:27:33.689013244 +0300
@@ -7,10 +7,10 @@
 )
 
 prefix = get_option('prefix')
-datadir = join_paths(prefix, get_option('datadir'))
-libdir = join_paths(prefix, get_option('libdir'))
-libexecdir = join_paths(prefix, get_option('libexecdir'))
-localedir = join_paths(prefix, get_option('localedir'))
+datadir = get_option('datadir')
+libdir = get_option('libdir')
+libexecdir = get_option('libexecdir')
+localedir = get_option('localedir')
 
 installed_tests_metadir = join_paths(datadir, 'installed-tests', meson.project_name())
 installed_tests_execdir = join_paths(libexecdir, 'installed-tests', meson.project_name())

